src/Esfa.Vacancy.UnitTests/Shared/Domain/GivenTrainingDetail.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using Esfa.Vacancy.Domain.Entities;
+using FluentAssertions;
+using NUnit.Framework;
+
+namespace Esfa.Vacancy.UnitTests.Shared.Domain
+{
+    [TestFixture]
+    public class GivenTrainingDetail
+    {
+        [Test]
+        public void AndDateIsBeforeToday_ThenExpiredShouldBeFalse()
+        {
+            var detail = new TrainingDetail { EffectiveTo = DateTime.Today.AddDays(-1) };
+            detail.HasExpired.Should().BeFalse();
+        }
+        [Test]
+        public void AndDateIsAfterToday_ThenExpiredShouldBeTrue()
+        {
+            var detail = new TrainingDetail { EffectiveTo = DateTime.Today.AddDays(1) };
+            detail.HasExpired.Should().BeTrue();
+        }
+        [Test]
+        public void AndDateIsToday_ThenExpiredShouldBeTrue()
+        {
+            var detail = new TrainingDetail { EffectiveTo = DateTime.Today };
+            detail.HasExpired.Should().BeFalse();
+        }
+    }
+}
```

Comment:
EffectiveTo date is not used at the moment, I included it thinking  we have to validate, this will be used in a later story so left it. Tests will come into play then.

Reply:
Ok